src/locales/en-US/translation.json

```
@@ -188,6 +188,19 @@
         }
       }
     },
+    "snowflake": {
+      "name": "snowflake",
+      "description": "Display the syntaxes for a specific Discord snowflake",
+      "responses": {
+        "invalidSnowflake": "The specified snowflake is invalid"
+      },
+      "options": {
+        "value": {
+          "name": "value",
+          "description": "The Discord snowflake"
```

I would avoid duplicating Discord here, besides redundancy the less mentions of it are in the code the more comfortable I feel about how clear it is that the bot is not affiliated with anything official
```suggestion
          "description": "The Snowflake number"
```